src/platform/packages/shared/kbn-unified-tabs/src/utils/get_tab_menu_items.tsx

```diff
@@ -34,13 +34,15 @@ const getTabMenuItem = ({
 
 export interface GetTabMenuItemsFnProps {
   tabsState: TabsState;
+  maxItemsCount: number | undefined;
```

Comment:
Should we unify `maxItemsCount: number | undefined` here and `maxItemsCount?: number` from `src/platform/packages/shared/kbn-unified-tabs/src/utils/manage_tabs.ts`?
Or we can pass `undefined` explicitly here?

Reply:
@akowalska622,

`maxItemsCount: number | undefined;` makes it a required param which can also be `undefined`.
`maxItemsCount?: number;` makes it an optional prop/param.

It depends on the context which one to use. For a component we can keep the prop as optional so consumers don't have to specify it. For internal utils like here it's better to keep it as a required param so we don't forget to pass it over.

Reply:
Cool! I'm fine with both, just wanted to ensure it's on purpose